ios/MaasCore/Sources/MaasTheme/Theme/Colors.swift

```diff
@@ -156,6 +156,25 @@ public struct GrayScalePalette {
     public let gray900: UIColor
 }
 
+public extension String {
+    func parseColor() -> Color {
+        .init(hex: self)
+    }
+}
+
+public extension Optional where Wrapped == String {
+    func parseColor() -> Color {
+        .init(hex: self)
+    }
+}
+
+public extension Color {
+    init(hex: String?) {
+        let uiColor: UIColor = ColorKt.toColor(hex, fallback: 0xFF_000000).color
+        self.init(uiColor)
+    }
+}
+
```

Comment:
I have a felling that we should create a separate `*.swift` for these and keep all extensions in one place. What do you think? 🤔

Reply:
Please do 🙏

Reply:
Kept `Color.init(hex: String?)` here as we want to keep the `UiColor.color` extension internal.